src/content/1.7/development/components/router-dispatcher.md

```diff
@@ -0,0 +1,64 @@
+---
+title: Router and Dispatcher component
+menuTitle: Router and Dispatcher
+---
+
+# Router and Dispatcher
+
+The legacy Dispatcher and the Symfony Router are in charge of parsing incoming HTTP requests and finding the right Controller to return HTTP response.
+
+### Front office routing
+
+The Front office controllers lie in the `controllers/front` directory.
+
+The Dispatcher match a given HTTP request with a Front Office controller.
+
+When you visit `/` on your shop you are being returned the Homepage, unless you have your shop installed in a subdirectory, then `/your_subdirectory` is your Homepage.
+
+What happened behind the scenes is that:
+
+1. Your HTTP request hit the `index.php` main routing file that relies on the _Dispatcher_
```

Comment:
```suggestion
1. Your HTTP request hits the `index.php` main routing file that relies on the _Dispatcher_
```

Reply:
No, this is past tense of `hit` verb, not present. And past tense does not require `s`
The full 1 - 2 - 3 list is written using past tense